features/dd-sdk-android-rum/src/main/java/com/datadog/android/rum/internal/instrumentation/gestures/GestureListenerCompat.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.rum.internal.instrumentation.gestures;
+
+import android.view.GestureDetector;
+import android.view.MotionEvent;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+public abstract class GestureListenerCompat implements GestureDetector.OnGestureListener {
```

Comment:
I know you opened a ticket at Google about this and I also updated that one, did they ever reply to you ? Nice trick anyway.

Reply:
No never got an answer ;D